auditlog/models/log.py

```diff
@@ -34,13 +38,31 @@ class AuditlogLogLine(models.Model):
     _description = "Auditlog - Log details (fields updated)"
 
     field_id = fields.Many2one(
-        'ir.model.fields', ondelete='cascade', string=u"Field", required=True)
+        'ir.model.fields', ondelete='set null', string=u"Field", required=True)
     log_id = fields.Many2one(
         'auditlog.log', string=u"Log", ondelete='cascade', index=True)
     old_value = fields.Text(u"Old Value")
     new_value = fields.Text(u"New Value")
     old_value_text = fields.Text(u"Old value Text")
     new_value_text = fields.Text(u"New value Text")
-    field_name = fields.Char(u"Technical name", related='field_id.name')
+    field_name = fields.Char(
+        string=u"Technical name", related='field_id.name', store=True)
     field_description = fields.Char(
         u"Description", related='field_id.field_description')
```

Comment:
store=True?

Reply:
Isn't a related field non-stored?

Reply:
Indeed you are correct, my bad. 
'''
:param store: whether the field is stored in database (boolean, by
            default ``False`` on computed fields)
'''

Reply:
I think we misunderstood each other. My suggestion was to add store=True to the definition of field_description like you did for field_name, but now I believe field_name has lost its `store` parameter. This would call for migration too of course.